earth_enterprise/src/fusion/fusionui/AssetDerivedImpl.h

```diff
@@ -134,11 +134,12 @@ void AssetDerived<Defs, FinalClass>::Init(bool re_init) {
 
   if (re_init == false) {
```

Comment:
what 're-init' means here ?

Reply:
This changed class 'AssetDerivedImpl' seems to be used in multiple places: imagery (resource, project), vector (resource, project), terrain (resource, project), database, KML project, Map layer, Map Project, Map database, Mercator image (resource, project), Mercator map database. We need to test all UI components that use those items to ensure that they work as designed.